### ISSUE TITLE
include TITLE in documentation/diagram

### ISSUE THREAD

**Issue by mamund** [koriym/app-state-diagram#10]
it would be great to be able to render the TITLE element (at the root of ALPS) on the output HTML and diagram.

also, maybe include the root level DOC.VALUE element on the HTML output?

thanks for a great tool. enjoying it very much.

**Comment by koriym** [comment]
>  be able to render the TITLE element (at the root of ALPS) on the output HTML

Sure. Also, the TITLE in the diagram would be nice?

> the root level DOC.VALUE 

Yes!

**Comment by koriym** [comment]
Like this?

```md
TodoMVC

--

# About 

* type: semantic
* doc.value: doc-value-here

--

home > asd > About

```

**Comment by koriym** [comment]
@mamund Is this fine?
https://koriym.github.io/app-state-diagram/

**Comment by mamund** [comment]
ah... yes.  very good.

Maybe on the next update you can add it to the top of the index page, too?

thanks for such a quick response on this feature!

cheers.

Mike Amundsen

*APIs, Microservices, and Digital Transformation*
7310 Turfway Rd. Suite 550, Florence, KY, 41042


*+1.859.372.6517*
https://zoom.us/j/8593726715
http://linkedin.com/in/mamund
http://g.mamund.com/meetme
http://training.amundsen.com
http://twitter.com/mamund

**Comment by koriym** [comment]
doc.value is already supported like this.

![image](https://user-images.githubusercontent.com/529021/98837762-b07afd80-2486-11eb-8d24-84fd2c1f90dc.png)

**Comment by mamund** [comment]
OK, i'll review my newly installed ASD and do some tests.

thanks.

Mike Amundsen

*APIs, Microservices, and Digital Transformation*
7310 Turfway Rd. Suite 550, Florence, KY, 41042


*+1.859.372.6517*
https://zoom.us/j/8593726715
http://linkedin.com/in/mamund
http://g.mamund.com/meetme
http://training.amundsen.com
http://twitter.com/mamund

**Comment by koriym** [comment]
"the top of the index page" done and released as `0.1.1`.
https://github.com/koriym/app-state-diagram/releases/tag/0.1.1

Also, I added how to update asd tool on README.

**Comment by mamund** [comment]
Thanks. Looks great.
